packages/components/psammead-media-player/src/Placeholder/index.jsx

```diff
@@ -12,15 +14,48 @@ const StyledPlaceholder = styled.div`
   height: 100%;
 `;
 
-const Placeholder = ({ onClick, src }) => (
-  <StyledPlaceholder onClick={onClick}>
-    <Image alt="Image Alt" src={src} />
-  </StyledPlaceholder>
-);
+const StyledPlayButton = styled(PlayButton)`
+  position: absolute;
+  bottom: 0;
+
+  ${StyledPlaceholder}:hover &,
+  ${StyledPlaceholder}:focus & {
+    background-color: ${C_POSTBOX};
+  }
+`;
+
+const Placeholder = ({ onClick, service, src, mediaInfo }) => {
+  const { title, datetime, duration, durationSpoken, type } = mediaInfo;
+  return (
+    <StyledPlaceholder onClick={onClick}>
+      <StyledPlayButton
+        title={title}
+        service={service}
+        onClick={event => {
+          event.stopPropagation();
```

Comment:
What is the effect of not having `stopPropagation` here?

Reply:
Am guessing what Simon was aiming for here with event.stopPropagation(); is to keep the onClick event from bubbling any further up into the DOM but I think what we really want is to use `event.preventDefault()` instead.

Reply:
but on second thought we do not need it as we only require the onClick to be called once

Reply:
As @RayNjeri mentions, this is to prevent the event from bubbling and triggering the parent handler a second time. Clicking both the placeholder or play button should trigger playback.

Reply:
Not sure I understand @simonsinclair, even from https://github.com/bbc/psammead/pull/2317#discussion_r335337769, can you explain more please?

Reply:
Hey @dr3, in the current PR, the play button's onClick calls the same handler as the parent (which needs to remain clickable), so using stopPropagation prevents the handler from being called twice.
Perhaps it would make more sense to use a no-op in the play button and let it bubble to the parent?